src/Symfony/Component/Lock/Tests/Store/BlockingStoreTestTrait.php

```diff
@@ -43,6 +43,13 @@ public function testBlockingLocks()
             return;
         }
 
+        // this function is not available in macOS. See https://bugs.php.net/bug.php?id=70708
+        if (!function_exists('pcntl_sigwaitinfo')) {
+            $this->markTestSkipped('pcntl_sigwaitinfo() PHP function is required.');
```

Comment:
this test function already have `@requires extension pcntl` annotation. for consistency you can use `@requires function pcntl_sigwaitinfo`

Reply:
Much better! Thanks.